metron-interface/metron-alerts/e2e/alerts-list/configure-table/configure-table.e2e-spec.ts

```diff
@@ -47,7 +47,7 @@ describe('metron-alerts configure table', function() {
     let newColNamesColumnConfig = [ 'score', 'timestamp', 'source:type', 'ip_src_addr', 'enrichments:geo:ip_dst_addr:country',
       'ip_dst_addr', 'host', 'alert_status', 'guid' ];
 
-    page.clearLocalStorage();
+    //page.clearLocalStorage();
```

Comment:
Can we remove this line instead of commenting it out?

Reply:
Done